### ISSUE TITLE
Middle mouse button create new note every time

### ISSUE THREAD

**Issue by dasrocky** [toeverything/AFFiNE#8438]
### What happened?

It may not be a bug but an intended feature, note creation with middle mouse button  that also allows you to pan around. Whenever the middle mouse button is pressed to move around the edgeless it creates a new note. Most of the time, it creates new notes with the last copied text. TBH, it is so annoying that I had to go back to FigmaJam temporarily. I am aware the shortcut `V` and `H` allow you to move around and select. It is easier and standard to move or pan around with the middle button (muscle memory too).

I come from Figma world. Most of the design tools and shortcuts they use are either standardized or they have it standardized. I also have used Penpot, blender etc, they all use middle mouse to move around.

### Distribution version

Linux

### What browsers are you seeing the problem on if you're using web version?

Chrome

### Are you self-hosting?

- [ ] Yes

### Anything else?

Here are the screenshot and Gifs:
![2024-10-0711-00-04](https://github.com/user-attachments/assets/86be0142-84fa-4874-a0d8-1bc8e1b43ceb)
![image](https://github.com/user-attachments/assets/5c98f033-e640-470d-bb4c-38fa8eea8a35)

**Comment by vascoferreira25** [comment]
This happens because it is Linux. By default Chrome and Firefox have a setting to paste on middle mouse button click. It would be nice if we could disable it in the settings.

**Comment by dasrocky** [comment]
> This happens because it is Linux. By default Chrome and Firefox have a setting to paste on middle mouse button click. It would be nice if we could disable it in the settings.

@vascoferreira25 You are right that is only a linux problem, I can use it well on my laptop. But my desktop PC, sadly. Is there a way to disable this from my side? Coz I don't face this issue in figma, lucid chart or even penpot. How is that not present in those mentioned app?
